sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/DisposeFromDisposeTest.cs

```diff
@@ -32,18 +33,19 @@ public class DisposeFromDisposeTest
     {
         [TestMethod]
         [TestCategory("Rule")]
-        public void DisposeFromDispose()
-        {
-            Verifier.VerifyAnalyzer(@"TestCases\DisposeFromDispose.cs", new DisposeFromDispose(), ParseOptionsHelper.FromCSharp8);
-        }
+        public void DisposeFromDispose_BeforeCSharp8() =>
+            Verifier.VerifyAnalyzer(@"TestCases\DisposeFromDispose.BeforeCSharp8.cs", new DisposeFromDispose(),
+                ImmutableArray.Create(new CSharpParseOptions(LanguageVersion.CSharp7_2)));
```

Comment:
This looks weird compared to method name. Two using statements can be removed afterwards.
```suggestion
                ParseOptionsHelper.BeforeCSharp8);
```
If there's a reason for 7_2-only, we should rename the method and the test case file.

Reply:
```
   Test method SonarAnalyzer.UnitTest.Rules.DisposeFromDisposeTest.DisposeFromDispose_BeforeCSharp8 threw exception:
SonarAnalyzer.UnitTest.TestFramework.UnexpectedDiagnosticException: CSharp5: Unexpected build error [CS8026]: Feature 'readonly structs' is not available in C# 5. Please use language version 7.2 or greater. on line 7
  Stack Trace:
```

I'll update the name

Reply:
Note: I don't see any reason in adding tests before C# 7.2, which already is 3 years old